test/call-expression.civet

```diff
@@ -132,11 +132,15 @@ describe "call-expression", ->
       ?.map((x) => x + 1)
     """
 
-    it "should not parse with !. on newline after plain identifier", ->
-      throws """
-        data
-        !.map (x) => x + 1
-      """
+    testCase """
+      !. on newline after plain identifier is a separate fn block
+      ---
+      data
+      !.map (x) => x + 1
```

Comment:
Took me a while to understand what this does... Unary operators with implicit `&` is a little crazy. But I guess it makes sense. (Though in this case doesn't make sense to negate an array :))

Reply:
Yeah, it's a little strange but falls out naturally from the existing rules.